### DISCUSSION TITLE
Setup

### DISCUSSION THREAD

**Discussion by fasakinhenry** [Adedoyin-Emmanuel/telebot#11] [Q&A] [answered]
How do I setup telebot as a newbie

**Comment by Adedoyin-Emmanuel** [comment, marked as answer]
## Get Started 🏃

1. Clone the repository

```bash
   git clone https://github.com/adedoyin-emmanuel/telebot
```

2. Install bot dependencies (You can use npm, yarn or bun)

```bash
   npm install

```

3. Get your bot token from **BotFather** <https://telegram.me/BotFather>.

4. Rename `.env.example` to `.env`

5. Copy your bot token to the BOT_TOKEN `.env` in the `.env` file.

6. Get your **Logsnag** **API_KEY** from <https://app.logsnag.com>

7. Paste your LogSnag **API_KEY** from <https://app.logsnag.com> to the LOGSNAG_TOKEN `.env` in the `.env` file.

8. Create a new project on **Logsnag**, copy the project name and then paste the project name in `LOGSNAG_PROJECT` `env` in the `.env` file.

9. Run the bot

```bash
   npm run dev
```
